malt/converters/slices.py

```diff
@@ -34,12 +34,21 @@ def _process_single_assignment(self, target, value):
     if not isinstance(target, gast.Subscript):
       return None
     s = target.slice
-    if isinstance(s, (gast.Tuple, gast.Slice)):
+    if isinstance(s, (gast.Tuple)):
       return None
 
     template = """
       target = ag__.set_item(target, key, item)
     """
+
+    if isinstance(s, (gast.Slice)):
+      template = template.replace("key", "slice(lower, upper)")
+      if s.upper is None:
+        return templates.replace(
+          template, target=target.value, lower=s.lower, upper=target.value, item=value)
```

Comment:
What is `target.value` in this case?

Reply:
It is variable's name. For example, in `x = setitem(x, Slice(y, z)`. `target.value = "x"`.

Reply:
Thanks, yeah the target being the bound seemed a bit weird